eng/scripts/resolve-package-info.ps1

```diff
@@ -0,0 +1,45 @@
+param (
+    [string] $ParameterTargetingStringSetting,
+    [string] $PackagePropertiesFolder,
+    [bool] $IncludeIndirect = $true
+)
+
+$setting = $ParameterTargetingStringSetting
+$targetingString = $ParameterTargetingStringSetting
+
+# users of internal service builds will set variable "BuildTargetingString" at runtime
```

Comment:
I assume you are referring to them setting it as a variable when they queue a build?